src/WebJobs.Script/Rpc/WebHostLanguageWorkerChannelManager.cs

```diff
@@ -56,8 +56,19 @@ private async Task<ILanguageWorkerChannel> InitializeLanguageWorkerChannel(strin
             {
                 languageWorkerChannel = _languageWorkerChannelFactory.CreateLanguageWorkerChannel(scriptRootPath, runtime, null, 0);
                 AddOrUpdateWorkerChannels(runtime, languageWorkerChannel);
-                await languageWorkerChannel.StartWorkerProcessAsync();
-                SetInitializedWorkerChannel(runtime, languageWorkerChannel);
+                await languageWorkerChannel.StartWorkerProcessAsync().ContinueWith(processStartTask =>
+                {
+                    if (processStartTask.Status == TaskStatus.RanToCompletion)
```

Comment:
nit: generally we do if (t.IsFaulted) { ... } else { ... }. A little easier to read